src/cljdoc/storage/api.clj

```diff
@@ -27,7 +27,7 @@
                             (:artifact-id version-entity)
                             [(:version version-entity)])
     (sqlite/import-api db-spec version-entity codox))
-  (import-doc [_ version-entity {:keys [doc-tree scm jar] :as version-data}]
+  (import-doc [_ version-entity {:keys [doc-tree scm jar config] :as version-data}]
```

Comment:
Looks like `config` and other keys are not being used. Why not skip destructuring?

Reply:
To remind myself what's in that map. Honestly it's probably a stupid thing to do as it will inevitably get out of date.